src/Tasks/ManifestUtil/mansign2.cs

```diff
@@ -707,40 +767,129 @@ private static void AuthenticodeSignLicenseDom(XmlDocument licenseDom, CmiManife
                                                              licenseDom.OuterXml + "</msrel:RelData>";
         }
 
-        private static void TimestampSignedLicenseDom(XmlDocument licenseDom, string timeStampUrl)
+        //
+        // ObtainRFC3161Timestamp
+        //
+        // This function is from mage.exe in .NET FX and is used to implement RFC 3161 timestamping.
```

Comment:
Can you link to the source please, even if it'll be a dead link for external contributors? That way internal folks at least can cross-reference.

Reply:
Mage.exe sources:
http://vstfdevdiv:8080/DevDiv2/DevDiv/_versionControl?path=%24%2FDevDiv%2FFX%2FFeature%2FNetFXDev1%2FPublic%2Finternal%2FNDP%2Finc%2Fmansign2.cs&line=1765&lineStyle=plain&lineEnd=1765&lineStartColumn=31&lineEndColumn=53